src/components/News/NewsContent/NewsContent.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FunctionComponent, useState } from "react";
+import { compareAsc, compareDesc } from "date-fns";
+import { Pagination } from "@govtechsg/tradetrust-ui-components";
+import { NewsTag, NewsSort, NewsSingle } from "./../types";
+import { NewsLink } from "./../NewsLink";
+import { NewsFilter } from "./../NewsFilter";
+
+export const newsPerPage = 12;
+
+export const NewsContent: FunctionComponent<{ allNews: NewsSingle[] }> = ({ allNews }) => {
+  const [searchStr, setSearchStr] = useState<string>("");
+  const [dropdownFilter, setDropdownFilter] = useState<NewsTag>();
+  const [dropdownSort, setDropdownSort] = useState<NewsSort>();
+
+  const filteredNews = allNews
+    .filter((news: NewsSingle) => news.attributes.title.toLowerCase().includes(searchStr.toLowerCase()))
```

Comment:
possible to extract the filter logics into its own function and name it?

something like:
```suggestion
const filterBySearchStr = React.useCallback((news: NewsSingle) => news.attributes.title.toLowerCase().includes(searchStr.toLowerCase(), [arr-of-deps...])
const filteredNews = allNews
    .filter(filterBySearchStr)
    ....
```

we can use useCallback for some memoisation

same goes for sort